consensus/view_change.go

```diff
@@ -368,6 +365,9 @@ func (consensus *Consensus) onViewChange(recvMsg *FBFTMessage) {
 	// already checked the length of SenderPubkeys in onViewChangeSanityCheck
 	senderKey := recvMsg.SenderPubkeys[0]
 
+	consensus.mutex.Lock()
```

Comment:
suggest moving these locks at the top of the func so that the logs about receiving the messages can be printed out in order of processing them. Otherwise, those logs are printed together at first and later on the messages are processed separately after acquiring locks. Basically not much performance concern to include the above in the lock. Consensus does that too.

Reply:
moved.